src/main/app/controller/courts/AddressController.ts

```diff
@@ -548,4 +550,14 @@ export class AddressController {
       }
     };
   }
+
+  private checkAddressesAreUnique(addresses: DisplayCourtAddresses) : string[]{
+    const errors: string[] = [];
```

Comment:
This looks a tad long :) 

In the future can use something like 

let hasDuplicate = arr.some((val, i) => arr.indexOf(val).getAddressLines() !== i.getAddressLines()); maybe? 

true = duplicated 
false = no duplicated

Reply:
Will consider in the future thanks